libtransmission/CMakeLists.txt

```diff
@@ -261,7 +261,8 @@ tr_target_compile_definitions_for_functions(${TR_NAME}
 
 target_include_directories(${TR_NAME}
     PRIVATE
-        ${CMAKE_CURRENT_BINARY_DIR}
+        ${CMAKE_CURRENT_SOURCE_DIR}/..
+        ${CMAKE_CURRENT_BINARY_DIR}/..
     INTERFACE
         ${CMAKE_CURRENT_SOURCE_DIR}/..
         ${CMAKE_CURRENT_BINARY_DIR}/..)
```

Comment:
```suggestion
    PUBLIC
        ${CMAKE_CURRENT_SOURCE_DIR}/..
        ${CMAKE_CURRENT_BINARY_DIR}/..)
```

Reply:
github won't let me merge your suggestion directly, so done manually in d948f0c